mapper/mapper_test.go

```diff
@@ -13,11 +15,42 @@ func TestUnitMapper(t *testing.T) {
 	ctx := context.Background()
 	cfg := config.Config{}
 	mdl := model.Page{}
+	req := httptest.NewRequest("", "/", nil)
 
 	Convey("test filter flex overview maps correctly", t, func() {
-		m := CreateFilterFlexOverview(ctx, mdl, cfg)
+		m := CreateFilterFlexOverview(ctx, req, mdl, cfg)
 		So(m.BetaBannerEnabled, ShouldBeTrue)
 		So(m.Type, ShouldEqual, "filter-flex-overview")
-		So(m.Metadata.Title, ShouldEqual, "Filter flex overview")
+		So(m.Metadata.Title, ShouldEqual, "Review changes")
+	})
+
+	Convey("test create selector maps correctly", t, func() {
+		m := CreateSelector(ctx, req, mdl, cfg, "dimensionName")
+		So(m.BetaBannerEnabled, ShouldBeTrue)
+		So(m.Type, ShouldEqual, "filter-flex-selector")
+		So(m.Metadata.Title, ShouldEqual, "DimensionName")
+	})
+}
+
+func TestUnitMapCookiesPreferences(t *testing.T) {
+	req := httptest.NewRequest("", "/", nil)
+	pageModel := model.Page{
+		CookiesPreferencesSet: false,
+		CookiesPolicy: model.CookiesPolicy{
+			Essential: false,
+			Usage:     false,
+		},
+	}
+
+	Convey("maps cookies preferences cookie data to page model correctly", t, func() {
+		So(pageModel.CookiesPreferencesSet, ShouldBeFalse)
+		So(pageModel.CookiesPolicy.Essential, ShouldBeFalse)
+		So(pageModel.CookiesPolicy.Usage, ShouldBeFalse)
+		req.AddCookie(&http.Cookie{Name: "cookies_preferences_set", Value: "true"})
```

Comment:
Should this be a separate Convey statement? One for true and one for false? Not really fussed about it but just wondering.

Reply:
TBH I copied it from the [dp-frontend-dataset-controller](https://github.com/ONSdigital/dp-frontend-dataset-controller/blob/3e8d60ecc5f52dd79847e416b3a3a660cd690c8c/mapper/mapper_test.go#L512)